fidesctl/src/fidesapi/main.py

```diff
@@ -58,9 +79,20 @@ async def db_action(action: DBActions) -> Dict:
 
 def start_webserver() -> None:
     "Run the webserver."
-    uvicorn.run(app, host="0.0.0.0", port=8080)
+    server = Server(Config(app, host="0.0.0.0", port=8080, log_level=logging.WARNING))
```

Comment:
There is an outstanding issue where uvicorn continues to swallow server logs. This might be mitigated by passing a `log_config` parameter here, but I can't find any useful documentation to suggest what we should pass. I was able to find [the default config](https://github.com/encode/uvicorn/blob/65ec8d132c37a8338a2f495fa1be9f14bd4368d9/uvicorn/config.py#L80), but it looks like `disable_existing_loggers` is already set to `False` 🤔

Reply:
noted, i think the logs are looking good though, so I'm ok to ignore this for now